assets/src/blocks/CSSVariablesToInlineText.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Converts an object with CSS Variables to inline text,
+ * for example, to use in a cssText property.
+ *
+ * @param {object} CSSVariables
+ * @returns {string}
+ */
+export const CSSVariablesToInlineText = CSSVariables => {
```

Comment:
Sorry to continue on the name nit picking :) But the inline part is not really accurate, you could as write the result of this function to a css file with a `::root {}` selector. Maybe `toDeclarations = CssVariables => {}` makes sense here?